src/Sylius/Behat/Context/Transform/CurrencyContext.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Transform;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Currency\Converter\CurrencyConverterInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class CurrencyContext implements Context
+{
+    /**
+     * @var CurrencyConverterInterface
+     */
+    private $currencyConverter;
+
+    /**
+     * @var RepositoryInterface
+     */
+    private $currencyRepository;
+
+    /**
+     * @param CurrencyConverterInterface $currencyConverter
+     * @param RepositoryInterface $currencyRepository
+     */
+    public function __construct(
+        CurrencyConverterInterface $currencyConverter,
+        RepositoryInterface $currencyRepository
+    ) {
+        $this->currencyConverter = $currencyConverter;
+        $this->currencyRepository = $currencyRepository;
+    }
+
+    /**
+     * @Transform /^currency "([^"]+)"$/
+     */
+    public function getCurrencyByName($currencyName)
+    {
+        $currencyCode = $this->currencyConverter->convertToCode($currencyName);
```

Comment:
See this interface: https://github.com/Sylius/Currency/blob/master/Converter/CurrencyConverterInterface.php, convertToCode is not there.